docker/shadow-hand/indigo/base/Dockerfile

```diff
@@ -4,15 +4,17 @@ LABEL Description="This ROS Indigo image contains Shadow's dexterous hand softwa
 
 ENV remote_shell_script="https://raw.githubusercontent.com/shadow-robot/sr-build-tools/$toolset_branch/bin/setup_dev_machine"
 
-RUN echo "Fixing file system" && \
-    mkdir -p /etc/apt/sources.list.d && \
+RUN echo "Fixing file system" && mkdir -p /etc/apt/sources.list.d && \
     rm -f /etc/apt/sources.list.d/ros-latest.list && \
 
     echo "Running one-liner" && \
     wget -O /tmp/oneliner "$( echo "$remote_shell_script" | sed 's/#/%23/g' )" && \
     chmod 755 /tmp/oneliner && \
     gosu $MY_USERNAME /tmp/oneliner -b "$toolset_branch" -w /home/user/projects/shadow_robot/base  && \
 
+    echo "Installing production tools" && \
+    curl https://raw.githubusercontent.com/shadow-robot/sr-build-tools/$(echo $toolset_branch | sed 's/#/%23/g')/bin/install-production-tools.sh | bash && \
```

Comment:
please use this approach for one liner call http://shadow-robot.readthedocs.io/en/latest/generated/shadow_robot/INSTALL.html#ros-kinetic